### PR TITLE
Add release-0.9 notes

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

File: content/en/community/releases.md [hunk 1]
```diff
@@ -5,6 +5,32 @@ weight: -20
 
 Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.8.0`. Continue reading to view upcoming releases:
 
+## `v0.9.0`, 21st, October 2022
+
+Open Cluster Management team is proud to annouce the release of OCM v0.9.0! Here are some main features included in this release:
```
Comment (dhaiducek) [inline content/en/community/releases.md:10]:
```suggestion
Open Cluster Management team is proud to announce the release of OCM v0.9.0! Here are some main features included in this release:
```

File: content/en/community/releases.md [hunk 2]
```diff
@@ -5,6 +5,32 @@ weight: -20
 
 Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.8.0`. Continue reading to view upcoming releases:
```
Comment (dhaiducek) [inline content/en/community/releases.md:6]:
```suggestion
Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.9.0`. Continue reading to view upcoming releases:
```

File: content/en/community/releases.md [hunk 3]
```diff
@@ -5,6 +5,32 @@ weight: -20
...
+- Support referencing the AddOn configuration with AddOn APIs 
+  For some add-ons, they want to run with configuration, we enhance the add-on APIs to support reference add-on configuration, and in add-on framework, we support to trigger re-rendering the add-on deployment if its configuration is changed
+- Allow Targeting Specific Services within Managed Clusters
+  The cluster-proxy add-on supports the exposure of services from within managed clusters to hub clusters, even across Virtual Private Clouds. Originally all traffic was routed through the Kubernetes API server on each managed cluster, increasing load on the node hosting the API server. Now the proxy agent  add-on supports specifying a particular target service within a cluster, allowing for better load balancing of requests made by hub clusters and more granular control of what resources/APIs are exposed to hub clusters.
+- Upgraded ManagedClusterSet API to v1beta2
+  Update the ClusterSet API and gradually remove legacy custom resources, as well as allow for transformation of legacy resources into analogous v1beta2 resources. v1alpha1 APIs are removed.
```
Comment (dhaiducek) [inline content/en/community/releases.md:19]:
```suggestion
  Update the ClusterSet API and gradually remove legacy custom resources, as well as allow for transformation of legacy resources into analogous v1beta2 resources. v1alpha1 APIs are removed.
- Consolidate the policy add-on template, status, and spec synchronization controllers into a single repository, [governance-policy-framework-addon](https://github.com/open-cluster-management-io/governance-policy-framework-addon)
```

File: content/en/community/releases.md [hunk 4]
```diff
@@ -5,6 +5,32 @@ weight: -20
...
+### Core components
+- registration v0.9.0 [changelog](https://github.com/open-cluster-management-io/registration/blob/v0.9.0/CHANGELOG/CHANGELOG-v0.9.md)
+- work v0.9.0 [changelog](https://github.com/open-cluster-management-io/work/blob/v0.9.0/CHANGELOG/CHANGELOG-v0.9.md)
+- placement v0.9.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.9.0/CHANGELOG/CHANGELOG-v0.9.md)
+- addon-framework v0.5.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.5.0/CHANGELOG/CHANGELOG-v0.5.md)
+- registration-operator v0.9.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.9.0/CHANGELOG/CHANGELOG-v0.9.md)
+
+### Addons
+
+TBD
```
Comment (dhaiducek) [inline content/en/community/releases.md:30]:
```suggestion
- config-policy-controller v0.9.0 [changelog](https://github.com/open-cluster-management-io/config-policy-controller/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
- governance-policy-framework-addon v0.9.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-framework-addon/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
- governance-policy-propagator v0.9.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-propagator/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
- governance-policy-addon-controller v0.9.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-addon-controller/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
FYI @mprahl

File: content/en/community/releases.md [hunk 5]
```diff
@@ -3,7 +3,36 @@ title: Releases
...
+- Allow Targeting Specific Services within Managed Clusters
+  The cluster-proxy add-on supports the exposure of services from within managed clusters to hub clusters, even across Virtual Private Clouds. Originally all traffic was routed through the Kubernetes API server on each managed cluster, increasing load on the node hosting the API server. Now the proxy agent  add-on supports specifying a particular target service within a cluster, allowing for better load balancing of requests made by hub clusters and more granular control of what resources/APIs are exposed to hub clusters.
+- Upgraded ManagedClusterSet API to v1beta2
+  Update the ClusterSet API and gradually remove legacy custom resources, as well as allow for transformation of legacy resources into analogous v1beta2 resources. v1alpha1 APIs are removed.
+- Consolidate the policy add-on template, status, and spec synchronization controllers into a single repository,   [governance-policy-framework-addon](https://github.com/open-cluster-management-io/governance-policy-framework-addon)
```
Comment (dhaiducek) [inline content/en/community/releases.md:20]:
```suggestion
- Consolidate the policy add-on template, status, and spec synchronization controllers into a single repository, [governance-policy-framework-addon](https://github.com/open-cluster-management-io/governance-policy-framework-addon)
```

**Review by dhaiducek — COMMENTED** [review]
Thanks for the updates! One minor change to remove extra whitespace.

File: content/en/community/releases.md [hunk 6]
```diff
@@ -3,7 +3,36 @@ title: Releases
...
+- Allow Targeting Specific Services within Managed Clusters
+  The cluster-proxy add-on supports the exposure of services from within managed clusters to hub clusters, even across Virtual Private Clouds. Originally all traffic was routed through the Kubernetes API server on each managed cluster, increasing load on the node hosting the API server. Now the proxy agent  add-on supports specifying a particular target service within a cluster, allowing for better load balancing of requests made by hub clusters and more granular control of what resources/APIs are exposed to hub clusters.
+- Upgraded ManagedClusterSet API to v1beta2
+  Update the ClusterSet API and gradually remove legacy custom resources, as well as allow for transformation of legacy resources into analogous v1beta2 resources. v1alpha1 APIs are removed.
+- Consolidate the policy add-on template, status, and spec synchronization controllers into a single repository,  [governance-policy-framework-addon](https://github.com/open-cluster-management-io/governance-policy-framework-addon)
```
Comment (mikeshng) [inline content/en/community/releases.md:20]:
```suggestion
- Consolidate the policy add-on template, status, and spec synchronization controllers into a single repository, [governance-policy-framework-addon](https://github.com/open-cluster-management-io/governance-policy-framework-addon)
- Application add-on is now able to expose custom Prometheus metrics via the Git subscription. See the [metric documentation](https://github.com/open-cluster-management-io/multicloud-operators-subscription/blob/v0.9.0/docs/metrics.md) for more details.
```

File: content/en/community/releases.md [hunk 7]
```diff
@@ -3,7 +3,36 @@ title: Releases
...
+- placement v0.9.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.9.0/CHANGELOG/CHANGELOG-v0.9.md)
+- addon-framework v0.5.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.5.0/CHANGELOG/CHANGELOG-v0.5.md)
+- registration-operator v0.9.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.9.0/CHANGELOG/CHANGELOG-v0.9.md)
+
+### Addons
+- config-policy-controller v0.9.0 [changelog](https://github.com/open-cluster-management-io/config-policy-controller/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
+- governance-policy-framework-addon v0.9.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-framework-addon/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
+- governance-policy-propagator v0.9.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-propagator/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
+- governance-policy-addon-controller v0.9.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-addon-controller/blob/main/CHANGELOG/CHANGELOG-v0.9.0.md)
+
```
Comment (mikeshng) [inline content/en/community/releases.md:34]:
```suggestion
- multicloud-operators-subscription v0.9.0 [release note](https://github.com/open-cluster-management-io/multicloud-operators-subscription/releases/tag/v0.9.0)
- multicloud-operators-channel v0.9.0 [release note](https://github.com/open-cluster-management-io/multicloud-operators-channel/releases/tag/v0.9.0)
- multicloud-integrations v0.9.0 [release note](https://github.com/open-cluster-management-io/multicloud-integrations/releases/tag/v0.9.0)
```

**Review by mikeshng — COMMENTED** [review]
Added app-addon notes. FYI @xiangjingli

**Comment by mikeshng** [comment]
https://www.cncf.io/blog/2022/10/31/open-cluster-management-november-2022-update/

CNCF blog post for reference.

**Comment by yue9944882** [comment]
LGTM

**Comment by mikeshng** [comment]
Not sure what happen I see a commit https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/commit/513cc83322122ab8f07922cc8febc29c371677f1 which contains the app addon changes but when I click Files Changes I don't see it.

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**